ui/src/table.tsx

```diff
@@ -486,16 +490,33 @@ const
           const epoch = Number(v)
           v = new Date(isNaN(epoch) ? v : epoch).toLocaleString()
         }
+
         if (col.key === primaryColumnKey) {
           const onClick = () => {
             wave.args[m.name] = [item.key as S]
             wave.push()
           }
-          return <Fluent.Link onClick={onClick}><TooltipWrapper>{v}</TooltipWrapper></Fluent.Link>
+          const align = col.align || 'left'
+          return (
+            <Fluent.Link onClick={onClick}>
+              <div style={{ display: 'flex', justifyContent: align, textAlign: align }}>
+                <TooltipWrapper>{v}</TooltipWrapper>
+              </div>
+            </Fluent.Link>)
```

Comment:
Is the `align` here necessary?

Reply:
Yes, if I remove this, then there are some values in the 1st column (I checked this using the `table.py` example) that are not correctly aligned.

For example, if I remove this styling, then the center alignment looks something like this on the 1st column (the red ones are incorrect) - 

<img width="347" alt="Screen Shot 2022-10-18 at 8 55 26 PM" src="https://user-images.githubusercontent.com/24620969/196594766-20060d9d-6227-41dc-8ce9-f3ed2049ea85.png">


Right alignment would look like this (the red ones are incorrect) - 

<img width="341" alt="Screen Shot 2022-10-18 at 8 55 42 PM" src="https://user-images.githubusercontent.com/24620969/196594803-4d9216ce-3a91-454f-ac86-789840bc0ba4.png">

Reply:
Got it. In that case, we probably need to do the same also for `return <TooltipWrapper>{v}</TooltipWrapper>`, right?

Reply:
I don't think we need to do that because once `<TooltipWrapper>{v}</TooltipWrapper>` is returned from `getCellComponent`, it is getting wrapped with the alignment logic in `onRenderItemColumn`. Let me know if I need to change this

Reply:
I see. The reason why it's needed here is that `Fluent.Link` has a style of `textAlign: 'left'` which overrides your flexbox solution. I fixed it there instead of adding an additional wrapper.

The `return <TooltipWrapper>{v}</TooltipWrapper>` has no such style thus it's not necessary as you correctly pointed out. Thanks!